chajda/tsquery/augments.py

```diff
@@ -205,35 +311,38 @@ def augments_fasttext(lang, word, config=Config(), n=5, annoy=True):
     # >>> augments_fasttext('es','escuela', n=5)
     # ['escuelala', 'academia', 'universidad', 'laescuela']
     '''
-    # download and load the fasttext model if it's not already loaded
-    try:
-        fasttext_models[lang]
-    except KeyError:
-        load_fasttext_model(lang)
-    
+
     # augments_fasttext defaults to using the annoy library to find nearest neighbors,
     # if annoy==False is passed into the function, then the fasttext library will be used.
     if annoy:
 
-        # populate and load annoy index with vectors from fasttext model if not already populated
-        create_annoy_index_if_needed(lang)
+        # populate and load AnnoyIndex with vectors from fasttext model if not already populated
+        load_annoy(lang)
 
         # find the most similar words using annoy library
-        n_nearest_neighbor_indices = annoy_indices[lang].get_nns_by_vector(fasttext_models[lang][word], n)
-        n_nearest_neighbors = [fasttext_models[lang].words[i] for i in n_nearest_neighbor_indices]
+        n_nearest_neighbor_indices = annoy_indices[lang].index.get_nns_by_item(annoy_indices[lang].index_for_word[word], n)
+        n_nearest_neighbors = [ annoy_indices[lang].word_at_index[f"{i}"] for i in n_nearest_neighbor_indices ]
```

Comment:
s/`f"{i}"`/`str(i)`

f-strings should only be used on more complicated strings

Reply:
it's a bit confusing that you're using the word "index" to refer to two different things in this code;  it refers both to the index as created by the annoy library (which is a large data structure) and the index into a list of words (which is just an int)

maybe renaming all latter used of the word index to "pos" might make the code more clear?

I'm not 100% that this change needs to be made if you don't think it makes it more clear, but you should at least be aware of the confusion that this code is likely to generate to readers and try to avoid these situations in the future

Reply:
I agree that this change would make things more clear. I'll go ahead and change all latter use of 'index' to 'pos'